docs/semgrep-supply-chain/malicious-dependencies.md

```diff
@@ -0,0 +1,69 @@
+---
+slug: malicious-dependencies
+title: Detect and remove malicious dependencies
+hide_title: true
+description: Learn how Semgrep detects malicious dependencies and enable malicious dependency detection in your Supply Chain scans.
+tags:
+  - Semgrep Supply Chain
+---
+
+# Detect and remove malicious dependencies
+
+:::info
+This feature is in private beta. To join, reach out to [support](/support).
+:::
+
+**Malicious dependencies** are intentionally dangerous packages, designed to compromise systems, unlike vulnerable dependencies, which are accidentally risky. These threats include packages that have always been malicious, such as typo-squatting attacks, or packages that become malicious after an attacker compromises a maintainer or injects harmful code. In some of these cases, the maliciousness of a package affects a specific range of versions.
+
+Semgrep is able to detect malicious dependencies in your projects and in pull requests (PRs) or merge requests (MRs).
+
+This feature is enabled after opting in to the beta program.
+
+## Supported package managers and sources of information
+
+Semgrep ingests data from Open Source Vulnerabilities ([<i class="fas fa-external-link fa-xs"></i>  osv.dev](https://osv.dev/)).
+
+The following table lists the languages for which Supply Chain can detect malicious dependencies.
+
+| Language | Package manager or ecosystem |
+| :---- | :---- |
+| C\# | NuGet |
+| Dart | Pub |
+| Elixir | Hex |
+| Go | `go.mod` |
+| Java | Maven |
+| Python | PyPi |
+| Ruby | RubyGems |
+| Scala | Maven |
+| Swift | SwiftPM |
+| PHP | \-- |
+| Rust | \-- |
+
+## About malicious dependency findings
+
+Malicious dependency findings are treated as **critical severity** findings.
```

Comment:
There was a discussion about this also being "always reachable" but no further confirmation.